### PR TITLE
Feature: Duplicate openpype instance operator

### PR DESCRIPTION
**Pull request by Tilix4**
## Brief description
- Duplicated with `.xxx` extension.
- Changed the UI of instance creation.

## Testing notes:
1. Create an instance
2. In context menu, duplicate the instance

### CODE REVIEW

File: openpype/hosts/blender/api/ops.py [hunk 1]
```diff
@@ -774,6 +773,52 @@ def execute(self, context):
...
+
+        # Change name
+        split_name = active_instance.name.rsplit(".", 1)
+        if len(split_name) > 1:
+            basename, number = split_name
+            extension = str(int(number) + 1).zfill(3)
+        else:
+            basename = split_name[0]
+            extension = "001"
+        new_instance.name = f"{basename}.{extension}"
```
Comment (kaamaurice) [inline openpype/hosts/blender/api/ops.py]: Numbered extension doesn't work great. We should do this in a while loop with incrementation and check of existing.
Capture d’écran 2023-02-02 155912

**Review by kaamaurice — CHANGES_REQUESTED** [review]

File: openpype/hosts/blender/api/ops.py (hunk 1 above)
Comment (Tilix4) [reply openpype/hosts/blender/api/ops.py]: I rewrote more, it seems to me more accurate and reliable. You'll tell me.

File: openpype/hosts/blender/api/ops.py [hunk 2]
```diff
@@ -771,6 +777,107 @@ def execute(self, context):
...
+
+        # Get active instance
+        op_instances = context.scene.openpype_instances
+        active_instance = op_instances[
+            context.scene.openpype_instance_active_index
+        ]
+
+        # Guess basename and extension
+        split_name = active_instance.name.rsplit(".", 1)
+        if len(split_name) > 1:
```
Comment (kaamaurice) [inline openpype/hosts/blender/api/ops.py]:
```suggestion
        if len(split_name) > 1 and split_name[1].isdigit():
```

**Review by kaamaurice — CHANGES_REQUESTED** [review]
Just add a safe check if instance name had "." but no digit

**Review by kaamaurice — APPROVED** [review]
LGTM

**Pull request merged**